index.html

```diff
@@ -495,6 +495,265 @@ <h2>
         </p>
       </section>
     </section>
+    <section>
+      <h2>
+        <code><dfn>user_preferences</dfn></code> member
+      </h2>
+      <p>
+        The `user_preferences` member of the [=application manifest=] is an
+        object that can be used to override values of manfiest members depending
+        on which user preferences are set. It has the following members:
+      </p>
+      <ul>
+        <li>[=user_preferences/color_scheme=]
+        </li>
+      </ul>
+      <p class="note">
+        This list of members is expected to expand in the future to include
+        other <a data-cite="mediaqueries-5/#mf-user-preferences">user 
```

Comment:
Try to avoid hard links to other specs... better to file a bug on the CSS spec to export the terms (bonus points for sending a PR to export the terms needed!🏆)